tests/runner.py

```diff
@@ -108,9 +108,10 @@ def parse_args():
     try:
         test_result = test_runner.run(tests)
         print_results(results, discovery_time, thresh, "tests finished")
+        sys.exit(not test_result.wasSuccessful())
```

Comment:
a quick feature request here: would be nice to have the `failfast` option to `runner.py`